packages/ripple-nuxt-tide/README.md

```diff
@@ -172,7 +172,7 @@ Icon.register({
 ```
 
 Update the name of the icon - this will be used for symbol prop later, `width`, `height`, `paths` and `polygons` accordingly.
-Add this to `assets/rpl_icon/index.js`
+Add this to `assets/ripple_icon/index.js`
```

Comment:
Hey @rubyllamadora `ripple-icon` instead of `ripple_icon`? Where is the code to load it? I am afraid we need a test for it as well.

Reply:
`ripple-icon`!. I might just revert this README update as this won't be rolled out to clients yet? It will still be svg files for them until we convert them to the new JS format.

Reply:
The example I added is inside `basic-examples` for enabled customIcon. I have JS files created in my local only as I think it is irrelevant to commit it for now.  We still need an extra script to process in replacement for `addIconsToLibrary()` once we roll this out.